README.md

```diff
@@ -17,14 +17,10 @@ This format can be used with [fastjet](http://fastjet.fr) directly.
 For 2022 data and MC **NanoAOD v11** according to the [XPOG](https://gitlab.cern.ch/cms-nanoAOD/nanoaod-doc/-/wikis/Releases/NanoAODv11) and [PPD](https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun3Analysis) recommendations:
 
 ```
-cmsrel CMSSW_12_6_0_patch1
-cd CMSSW_12_6_0_patch1/src
+cmsrel CMSSW_12_6_5
```

Comment:
any reason why switching to 12_6_5 is needed?

Reply:
here followed the convention that Z in X_Y_Z are always updated to the latest